solutions/CSES Road Construction.mdx

```diff
@@ -54,9 +56,10 @@ int main () {
 
 <JavaSection>
 
-```java
-
+**Time Complexity:** $\mathcal{O}(N\log N)$
```

Comment:
Put this before "LanguageSection" I think it can be the reason why the build is not working.

Reply:
I think this is fine, it seems that the prettier check is failing.

Reply:
Do you know how to solve that?

Reply:
ehh just ignore it, I think I'm going to get rid of the prettier check tbh. It's causing a lot more trouble than it's actually helping imo